README.md

```diff
@@ -1,3 +1,8 @@
+# THIS REPOSITORY HAS MOVED.
+
+Hobbits is now developed at https://github.com/deltap2p/hobbits. See you there!
+
```

Comment:
Can we change this to something more like:

"Hobbits has been adopted by the [DeltaP2P Foundation](https://github.com/deltap2p/hobbits) where it is now being maintained."